src/components/Avatar.vue

```diff
@@ -35,22 +38,24 @@ export default {
       const url = instance + '/remote.php/dav/avatars/' + this.userid + '/128.png'
       headers.append('Authorization', 'Bearer ' + this.getToken)
       headers.append('X-Requested-With', 'XMLHttpRequest')
-      fetch(url, { headers })
-        .then(response => {
-          if (response.ok) {
-            return response.blob()
-          }
-          throw new Error('Network response was not ok.')
-        })
-        .then(blob => {
-          this.loading = false
-          this.avatarSource = window.URL.createObjectURL(blob)
-        })
-        .catch(error => {
-          this.avatarSource = ''
-          this.loading = false
-          console.log('There has been a problem with your fetch operation: ', error.message)
-        })
+      window.setTimeout(() => {
+        fetch(url, { headers })
+          .then(response => {
+            if (response.ok) {
+              return response.blob()
+            }
+            throw new Error('Network response was not ok.')
+          })
+          .then(blob => {
+            this.loading = false
+            this.avatarSource = window.URL.createObjectURL(blob)
+          })
+          .catch(error => {
+            this.avatarSource = ''
+            this.loading = false
+            console.log('There has been a problem with your fetch operation: ', error.message)
```

Comment:
Let's change the log msg to `... with the image fetch operation` or something like that 😁